app/views/shared/_email_form.html.erb

```diff
@@ -48,7 +48,7 @@
   <% if current_user.blank? %>
     <div class="form-group row">
       <div class="col-sm-10 offset-sm-2">
-        <%= recaptcha_tags id: 'connection-form-recaptcha' %>
+        <%= recaptcha_tags id: 'connection-form-recaptcha', noscript: false %>
```

Comment:
Would you mind adding a comment that explains why we're removing noscript?

Reply:
Not at all, added.